servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -308,6 +305,7 @@ def _collect_change_requests(self, instance_info):
             'sysparm_limit': instance_info.change_request_process_limit,
             'sysparm_query': sysparm_query
         }
+        params = self._params_append_to_sysparm_query(instance_info.change_request_sysparm_query, params)
```

Comment:
I miss a test when collecting topology events which ensure that the `sysparm` is set correctly if we configure the query filtering

Reply:
added.